_docs/useful-links.md

```diff
@@ -29,7 +29,7 @@ The following is a list of links for data providers and web applications develop
 
   The IOOS Compliance Checker is a python based tool for data providers to check for completeness and community standard compliance of local or remote netCDF files against metadata standards. This web based tool can be used to check your GliderDAC netCDF files for compliance. Be sure to select **GliderDAC** in the list of available tests to run.
 
- - [**FTP script**](https://github.com/ioos/ioosngdac/blob/master/util/ncFtp2ngdac.pl)
+ - [**FTP script**](https://github.com/ioos/glider-dac/blob/master/util/ncFtp2ngdac.pl)
```

Comment:
@benjwadams @mwengren FYI - this .pl script includes data.ioos.us instead of gliders.ioos.us, and is linked in the User Documentation (https://ioos.github.io/glider-dac/useful-links.html).  Will this script be updated?  Or should we remove it?

Reply:
Update - we'll remove this from the docs.